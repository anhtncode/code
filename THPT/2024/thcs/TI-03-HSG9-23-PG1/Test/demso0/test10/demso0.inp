3
110221
356975
718205 
