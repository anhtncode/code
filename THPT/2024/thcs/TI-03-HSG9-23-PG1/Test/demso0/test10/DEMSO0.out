27552
89240
179547
